content/blog/2022/2022-12-13-dask-table.md

```diff
@@ -0,0 +1,34 @@
++++
+author = "Noah Novšak"
+date = "2022-12-13"
+draft = false
+title = "Orange you going to ask about dask"
+type = "blog"
+thumbImage = "/blog_img/2022/2022-12-13_dask_data.png"
+frontPageImage = "/blog_img/2022/2022-12-13_dask_data.png"
+blog = ["dask", "widgets"]
+shortExcerpt = "Preparing Orange to handle much more data."
```

Comment:
I'd like longer shortExcerpts (3 lines on the first page) and longExcepts (~5+ lines in the Blog list).

That reminds me, we should have some guidelines about this.